ios/App/App/capacitor.config.json

```diff
@@ -1,7 +1,10 @@
 {
-  "appId": "org.lunie.lunie",
-  "appName": "Lunie",
-  "bundledWebRuntime": false,
-  "npmClient": "yarn",
-  "webDir": "dist"
-}
+    "appId": "org.lunie.lunie",
+    "appName": "Lunie",
+    "bundledWebRuntime": false,
+    "npmClient": "yarn",
+    "webDir": "dist",
+    "server": {
+        "url": "http://localhost:9080/"
```

Comment:
ah or does this do use the served files? that would be neat

Reply:
yep, this is from what is served locally